eng/scripts/TypeSpec-Generate-Sdk.ps1

```diff
@@ -0,0 +1,35 @@
+[CmdletBinding()]
+param (
+  [Parameter(Position = 0)]
+  [ValidateNotNullOrEmpty()]
+  [string] $SdkRepoRootDirectory,
+  [Parameter(Position = 1)]
+  [string] $TypeSpecProjectDirectory = ".", # A directory of `tspconfig.yaml` or a remoteUrl of `tspconfig.yaml`
+  [Parameter(Position = 2)]
+  [string] $CommitHash,
+  [Parameter(Position = 3)]
+  [string] $RepoUrl
```

Comment:
can this be optional with a default of https://github.com/azure/azure-rest-api-specs?

Reply:
Yes, it's the default value to be used to create tsp-location.yaml when doesn't pass in this parameter.